Github Issue Reader/ViewController.swift

```diff
@@ -14,36 +14,73 @@ class ViewController: UIViewController {
     var containerView: UIStackView = {
         let view = UIStackView()
         view.axis = .vertical
+        view.alignment = .leading
+        view.distribution = .fill
         view.translatesAutoresizingMaskIntoConstraints = false
         return view
     }()
     
+    var gitLabel: UILabel = {
+        let label = UILabel()
+        label.text = "GitHub Issue Viewer"
+        label.font = UIFont.preferredFont(forTextStyle: .largeTitle)
+        return label
+    }()
+    
+    var gitSubLabel: UILabel = {
+        let label = UILabel()
+        label.text = "View issues in a repository:"
+        label.font = UIFont.preferredFont(forTextStyle: .title2)
+        label.textColor = UIColor.systemGray
+        return label
+    }()
+    
+    var orgLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Organization"
+        label.font = UIFont.preferredFont(forTextStyle: .title2)
+        return label
+    }()
+    
+    var repoLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Repository"
+        label.font = UIFont.preferredFont(forTextStyle: .title2)
+        return label
+    }()
+    
     var orgTextfield: UITextField = {
         let textfield = UITextField()
         textfield.placeholder = "Enter an organization"
+        textfield.borderStyle = .roundedRect
+        textfield.backgroundColor = .clear
         return textfield
     }()
     
     var repoTextfield: UITextField = {
         let textfield = UITextField()
         textfield.placeholder = "Enter a repository"
+        textfield.borderStyle = .roundedRect
+        textfield.backgroundColor = .clear
         return textfield
     }()
     
     lazy var submitButton: UIButton = {
         let button = UIButton()
+        var config = UIButton.Configuration.filled()
+        button.setTitle("View Issues", for: .normal)
+        button.setTitleColor(.white, for: .normal)
         button.tintColor = .black
```

Comment:
These 3 properties can be set using the config. As a general statement using the same system for something causes less weird bugs than if you mix and match.

```
config.title = "View Issues"
config.baseForegroundColor = .white
config.baseBackgroundColor = .black
```